.github/workflows/test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
       - name: Run tests
         run: |
-          nix run .#marlowe-integration-tests -- --strict
+          nix develop --command run-integration-tests --strict
```

Comment:
Won't this build the Nix dev shell in the test runner? We don't want to do that; just build the app.